### PR TITLE
config prow merge for GoogleCloudPlatform/netd

### PR DESCRIPTION
**Pull request by cadmuxe**

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -410,6 +410,18 @@ tide:
...
+    - lgtm
+    - approved
+    missingLabels:
+    - do-not-merge
+    - do-not-merge/blocked-paths
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+
   merge_method:
```
Comment (cadmuxe) [inline prow/config.yaml]: do I need to config the merge_method?

Comment (krzyzacy) [reply prow/config.yaml]: depend on if you want to squash or not :-)

Comment (cadmuxe) [reply prow/config.yaml]: k, I will following the default. 
Thanks

**Pull request merged**